### PR TITLE
47 table

### PR DESCRIPTION
**Pull request by boxadesign**

### CODE REVIEW

File: src/components/table/src/_table.scss [hunk 1]
```diff
@@ -0,0 +1,178 @@
+.table {
```
Comment (bameyrick) [inline src/components/table/src/_table.scss]: Potentially the child `.table__<thing>` could be `&__<thing>` which will reduce the amount of classnames outputted (e.g. `.table .table__caption {` would just be `.table__caption { `), but also specificity which could affect overriding things. 🤷🏻‍♂️

File: src/components/table/src/_table.scss [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      outline: 3px solid $color-focus;
+      outline-offset: 3px;
+    }
+    .table__header,
+    .table__cell {
+      @include mq(xs, m) {
+        white-space: nowrap;
+      }
+    }
+    .right-shadow,
```
Comment (bameyrick) [inline src/components/table/src/_table.scss]: Should these classes not be `table__right-shadow` and `table__left-shadow`?

File: src/components/table/src/_template.njk [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{% if params.scrollable %}
```
Comment (bameyrick) [inline src/components/table/src/_template.njk]: All of the hardcoded aria-labels etc should probably be parameterised to allow for internationalisation

File: src/components/table/index.njk [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- **Output** - This reduces the `font-size` to compact the table for basic tables which have many rows but only a few columns.
+
+### Numeric table
+{{
+    patternlibExample({"path": "components/table/examples/table-numeric/index.njk"})
+}}
+- **Class** - `table__header--numeric` + `table__cell--numeric`
+- **Output** - This aligns the content to the right for tables with all numeric values for best practice.
+
+### Reponsive table
```
Comment (jrbarnes9) [inline src/components/table/index.njk]: Typo!

File: src/components/table/src/_table.scss (hunk 1 above)
Comment (boxadesign) [reply src/components/table/src/_table.scss]: Yep you're right.

**Comment by bameyrick** [comment]
@boxadesign I already did?

**Pull request merged**